build.gradle.kts

```diff
@@ -11,7 +11,7 @@ buildscript {
         gradlePluginPortal()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:8.0.0")
+        classpath("com.android.tools.build:gradle:8.0.1")
```

Comment:
Please do not change versions in a feature PR unless required for the build to work.

Reply:
Removed non-related changes